src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs

```diff
@@ -4132,48 +4063,61 @@ void EmitSingleCharRepeater(RegexNode node, bool emitLengthChecksIfRequired = tr
                 else
                 {
                     // ReadOnlySpan<char> tmp = slice.Slice(sliceStaticPos, iterations);
-                    // for (int i = 0; i < tmp.Length; i++)
-                    // {
-                    //     TimeoutCheck();
```

Comment:
Judging from the old code, we actually weren't emitting the timeout check here, right?

Reply:
Correct.  We used to, and then we removed that as part of overhauling how/where we emit timeout checks... the comment was just stale.